### DISCUSSION TITLE
Imho not enough / good looking space in a pnp search webpart

### DISCUSSION THREAD

**Discussion by keptenkrems** [microsoft-search/pnp-modern-search#4041] [Ideas]
Hi everybody,

It is not a bug, it is more a thing I see when I use the pnp search webparts... for me there is in comparison to the standardwebparts not the same / not enough space between the ends in a section... the pictures I have attached will show the pnp modern search webpart with a Title in a section and I have a standard textwebpart in a section. The space between the next section is different.
Is there a way to fix or modify it?

![2024-10-11 15_55_15-MY-O – Homepage und 6 weitere Seiten - Geschäftlich – Microsoft​ Edge](https://github.com/user-attachments/assets/f257e80a-8bb3-4333-82c1-7e47d03b0c00)
![2024-10-11 15_55_53-MY-O – Homepage und 6 weitere Seiten - Geschäftlich – Microsoft​ Edge](https://github.com/user-attachments/assets/eaa0450a-fdc3-41ac-bea0-d6c4d3554a7f)

Also when I do not use the counting of the items. the space between the title and the result is not very smooth looking.
![2024-10-11 16_06_02-MY-O – Homepage und 11 weitere Seiten - Geschäftlich – Microsoft​ Edge](https://github.com/user-attachments/assets/c9c74f39-90c1-4e98-be8c-5aa684e0b5b2)

Thanks in advance
BR Tobias

**Comment by wobba** [comment]
If you can compare with more web parts and propose what the spacing should be it can be looked at. Or, don't use the web part title and add the title via the handlebar template instead which gives you full CSS control.

**Reply by keptenkrems** [reply]
Hi Mikael,
thank you for your response.
If I compare it with e.g. the news webpart... there are about 38 px from the top / bottom.
<img width="541" alt="image" src="https://github.com/user-attachments/assets/5b1e68ab-1ee7-452b-bbe9-efbce920ad9d">

If I compare it with other webpart, all of them differs... e.g. from a picture webpart, there are 33px from the top and 38 from the bottom... at a people webpart there are more then 50px from the top/bottom... 

I want an title, because in my example I will show faults and planned maintenance news/posts. If there are no planned news or posts to that topics, nothing should show up, so the section does not appear... so my plan...

how do you mean the solution with the handlebar template? I do know that much about the handlebar topics - I am still in that learning process.

BR and thanks for your answer.
Tobias

**Reply by wobba** [reply]
If OOB web parts differ on the space at the top I think it's better to add the title to the template and set your spacing there instead. Pick a template, then click the `{}` button on `Edit results template` and add html/css to some title text at the top of the template any way you want.

![image](https://github.com/user-attachments/assets/5e63690b-a9b6-40dc-8c2d-48e43de1f4f2)
